### DISCUSSION TITLE
Problem with select and textfield type date in material ui when using reset for default values

### DISCUSSION THREAD

**Discussion by arbrito** [react-hook-form/react-hook-form#7118] [Q&A]
Hi, if you could help me, I am having problems with the select and date type textfield when I am going to initialize them with the default values brought from the api, the "reset" works perfectly for the text type textfields, I am using ui and yup material for validations.

**Comment by bluebill1049** [comment]
provide a simple codesandbox

**Reply by arbrito** [reply]
Thank you very much @bluebill1049 , I have been stuck in this for many days.
https://codesandbox.io/s/client-k9g7e?file=/src/EditClient.js
